src/Magnum/ImGuiIntegration/Context.cpp

```diff
@@ -312,17 +314,17 @@ void Context::drawFrame() {
 
     ImGui::Render();
 
-    ImGuiIO& io = ImGui::GetIO();
-    const Vector2 fbSize = Vector2{io.DisplaySize}*Vector2{io.DisplayFramebufferScale};
-    if(!fbSize.product()) return;
-
     ImDrawData* drawData = ImGui::GetDrawData();
     CORRADE_INTERNAL_ASSERT(drawData); /* This is always valid after Render() */
-    drawData->ScaleClipRects(io.DisplayFramebufferScale);
+
+    const Vector2 fbSize = Vector2{drawData->DisplaySize}*Vector2{drawData->FramebufferScale};
```

Comment:
Hm, I can't tell if this is a good or a bad change as my knowledge of ImGui is rather bad :sweat_smile: How does `drawData->FramebufferScale` and `io.DisplayFramebufferScale` relate?

Cc: @pezcode, you're responsible for most of the code here I think, and I bet you know better than me :)

Reply:
From the ImGui comments:
```
ImVec2          DisplayPos;             // Top-left position of the viewport to render (== top-left of the orthogonal projection matrix to use) (== GetMainViewport()->Pos for the main viewport, == (0.0) in most single-viewport applications)
ImVec2          DisplaySize;            // Size of the viewport to render (== GetMainViewport()->Size for the main viewport, == io.DisplaySize in most single-viewport applications)
ImVec2          FramebufferScale;       // Amount of pixels for each unit of DisplaySize. Based on io.DisplayFramebufferScale. Generally (1,1) on normal display, (2,2) on OSX with Retina display.
```

Reply:
It's mainly to aid with multi-window rendering 😉

Reply:
I see, that makes sense then :)